src/adafruit_blinka/microcontroller/mcp2221/mcp2221.py

```diff
@@ -64,6 +94,46 @@ def __init__(self):
         for pin in range(4):
             self.gp_set_mode(pin, self.GP_GPIO)  # set to GPIO mode
             self.gpio_set_direction(pin, 1)  # set to INPUT
+        return self
+
+    @staticmethod
+    def available_paths(*, require_mcps=False):
+        """Return a list of paths of the currently available MCP2221s.
+
+        Raises a RuntimeError if *require_mcps* is True and there
+        are no MCP2221 connected.
+        """
+        mcps = hid.enumerate(MCP2221.VID, MCP2221.PID)
+        if require_mcps and not mcps:
+            raise RuntimeError('No MCP2221 detected.')
+        return [mcp["path"] for mcp in mcps]
+
+    @classmethod
+    def new_instance(cls):
```

Comment:
This method will try to find a new connected MCP that is not cached, by looping through the available devices and trying to open them.  This is convenient when there are multiple MCPs connected, since the user can just call `MCP2221.new_instance()` repeatedly to get an instance for each of them.

A few lines here are duplicated with the `if` above (lines 78-84), but a refactoring to remove the duplication doesn't seem worthy.

Reply:
This method is now gone and integrated into `__new__`.